docs/configuration/large_language_model.md

```diff
@@ -88,21 +88,66 @@ ludwig train ...
 
 ## Input Features
 
-Currently, the LLM model type only supports a single input feature of type `text`.
+Because of the way LLMs work, their input is always a single chunk of `text`. But this input can be 
```

Comment:
@sanjaydasgupta Please consider the following changes:
1) "their input is always a single chunk of `text`." -> "they accept exactly one input, which is some amount of `text`."
2) Remove "But": "This input can be [...etc...]".
3) Change "is immaterial, and is not" to "is immaterial; it is not".

Everything else looks great!  Thank you so much!

Reply:
Done!